src/lib-components/BlockHours.vue

```diff
@@ -6,6 +6,7 @@
             <!--div class="iframe-container"-->
             <iframe
                 id="the-iframe"
+                title="Hours for location"
```

Comment:
We can add a location name prop and pass it at the page level here https://github.com/UCLALibrary/library-website-nuxt/blob/189907695701cd69ed2b639726bbf8c60f14a3a7/pages/visit/locations/_slug.vue#L66